### ISSUE TITLE
Error with the following config... should not happen

### ISSUE THREAD

**Issue by xFragger** [daddy-cool/highlogger#5]
```
..../node_modules/highlogger/lib/highlogger.js:95
      config.debugKeys.exclude = CONFIG_DEFAULT.debugKeys.exclude;
                               ^

TypeError: Cannot assign to read only property 'exclude' of #<Object>
    at populateConfig (..../node_modules/highlogger/lib/highlogger.js:95:32)
```
when using this config
```
{ transporters: [ { type: 0 } ],
  debugKeys: { include: [ '*' ], exclude: null } }
```

**Comment by daddy-cool** [comment]
Can't reproduce with this config, looks to me more like you're passing a weird object with read-only attributes. Even then this shouldn't happen but I will look into it and it should be fixed in v1.8.0

**Comment by daddy-cool** [comment]
Please try again with v1.8.1
